curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64a234995c47cf02021508f1.md

```diff
@@ -0,0 +1,338 @@
+---
+id: 64a234995c47cf02021508f1
+title: Step 33
+challengeType: 0
+dashedName: step-33
+---
+
+# --description--
+
+just like step 31, you will use the `innerHTML` property on the `currentRoundRollsText` variable to set value of `rolls`. Do the same for `currentRoundText` variable to set value of `round`. This will reset the rolls and round to `0`
```

Comment:
```suggestion
Just like in previous steps, you will use the `innerHTML` property on the `currentRoundRollsText` variable to set the value of `rolls`. Do the same for `currentRoundText` variable to set the value of `round`. This will reset the rolls and round to `0`
```